### DISCUSSION TITLE
Platform-specific sections in config file?

### DISCUSSION THREAD

**Discussion by emendelson** [joncampbell123/dosbox-x#4678] [Ideas]
A discussion elsewhere about PostScript support in macOS suggests a feature for the distant future - platform-specific instructions in a config file. For example, the `[parallel]` section might include lines that apply only if the host is macOS, overriding other lines that would be used in other systems, something like this:

```
[parallel]
parallel1: .....
[[macosx]]
parallel1:....
[[/macosx]]
parallel2:...
```
I used `[[macosx]]` and `[[/macosx]]` to indicate the start and end of the macOS-specific sections, but of course anything else could do the job.

This could reduce the effort required of individual users to adjust to their host systems.

**Comment by rderooy** [comment]
I'm not a fan. To me it adds a lot of complexity and potential confusion for little to no benefit.

**Comment by emendelson** [comment]
And the more I think about it, the less I think it's a good idea. If I could delete this thread, I could...
